game_engine.properties

```diff
@@ -4,15 +4,3 @@
 ## first.
 
 engine_version = 1.9.0.0
-
-## URL where we will download the list of available maps
-map_list_file = https://raw.githubusercontent.com/triplea-game/triplea/master/triplea_maps.yaml?raw=true
-## Example that would use a local file config file, useful for map building or development
-# map_list_file = triplea_maps.yaml
-
-
-## Location where we download latest lobby description file, this tells us where the latest lobby is located
-lobby_properties_file_url = https://raw.githubusercontent.com/triplea-game/triplea/master/lobby_server.yaml?raw=true
```

Comment:
Sorry I did not quite convey the full idea. There are no changes for anything other than two properties removed. We currently have the engine version, that is the only thing left in the game_engine.properties file.

For the feature toggles, IMO we should include them with on/off setting so it's easier to update. Knowing what the right property key is a small challenge.  Regardless, that capability is still there. It's just that the only thing that remains after this update in the `game_engine.properties` file after this update is version number.

For the two URLs, the backup URl was effectively a hack for us to override the lobby URL. Hence, for those two fields, after removing how the backup URL was done and allowing the URLs to be updated in settings UI, the initial value may as well come from code (hence the props removal).

IMO the JFX and proxy feature toggles would be well moved to the new settings UI and mimic the beta feature flag. By adding a couple enums you would get a radio button that could turn those features on/off. game_engine.properties is though not a bad answer for props that need to be enabled on startup.